python/redmonster/application/zfind.py

```diff
@@ -90,9 +90,9 @@ def set_templates(self, templates=None, zmin=None, zmax=None, npoly=None, npixst
         if type(self.templates) is str:
             try: self.templates = [self.templates]
             except:
-                print 'Templates not a list and unable to convert to list!'
+                print('Templates not a list and unable to convert to list!')
                 sys.exit(1)
-        if type(self.templates) is list: self.templates = map(str, self.templates)
+        if type(self.templates) is list: self.templates = list(map(str, self.templates))
```

Comment:
That section of code is a potential can of worms so I'm inclined to leave it alone, especially because I'd probably also have to fix it in zfind2.py, so it's double the work.